tools/build_defs/cc_linking_util.bzl

```diff
@@ -2,24 +2,56 @@
 """
```

Comment:
I'd rename this file, it's not strictly speaking only dealing with linking, right? Wdyt about cc_toolchain_util?

Reply:
Done.